src/base/bittorrent/session.cpp

```diff
@@ -2351,86 +2585,90 @@ void Session::handleStateUpdateAlert(libt::state_update_alert *p)
     emit torrentsUpdated();
 }
 
-bool readFile(const QString &path, QByteArray &buf)
-{
-    QFile file(path);
-    if (!file.open(QIODevice::ReadOnly)) {
-        qDebug("Cannot read file %s: %s", qPrintable(path), qPrintable(file.errorString()));
-        return false;
-    }
-
-    buf = file.readAll();
-    return true;
-}
-
-bool loadTorrentResumeData(const QByteArray &data, AddTorrentData &out, int &prio,  MagnetUri &magnetUri)
+namespace
 {
-    out = AddTorrentData();
-    out.resumed = true;
-    out.skipChecking = false;
-
-    libt::lazy_entry fast;
-    libt::error_code ec;
-    libt::lazy_bdecode(data.constData(), data.constData() + data.size(), fast, ec);
-    if (ec || (fast.type() != libt::lazy_entry::dict_t)) return false;
+    bool readFile(const QString &path, QByteArray &buf)
+    {
+        QFile file(path);
+        if (!file.open(QIODevice::ReadOnly)) {
+            qDebug("Cannot read file %s: %s", qPrintable(path), qPrintable(file.errorString()));
+            return false;
+        }
 
-    out.savePath = Utils::Fs::fromNativePath(Utils::String::fromStdString(fast.dict_find_string_value("qBt-savePath")));
-    if (out.savePath.isEmpty()) {
-        Logger::instance()->addMessage("Empty torrent save path was loaded from .fastresume file! Using default one...", Log::WARNING);
-        out.savePath = Preferences::instance()->getSavePath();
+        buf = file.readAll();
+        return true;
     }
 
-    out.ratioLimit = Utils::String::fromStdString(fast.dict_find_string_value("qBt-ratioLimit")).toDouble();
-    out.label = Utils::String::fromStdString(fast.dict_find_string_value("qBt-label"));
-    out.name = Utils::String::fromStdString(fast.dict_find_string_value("qBt-name"));
-    out.hasSeedStatus = fast.dict_find_int_value("qBt-seedStatus");
-    out.disableTempPath = fast.dict_find_int_value("qBt-tempPathDisabled");
-
-    magnetUri = MagnetUri(Utils::String::fromStdString(fast.dict_find_string_value("qBt-magnetUri")));
-    out.addPaused = fast.dict_find_int_value("qBt-paused");
-    out.addForced = fast.dict_find_int_value("qBt-forced");
-
-    prio = fast.dict_find_int_value("qBt-queuePosition");
-
-    return true;
-}
+    bool loadTorrentResumeData(const QByteArray &data, AddTorrentData &out, int &prio,  MagnetUri &magnetUri)
```

Comment:
Wouldn't it be more easier for a reader if you either rename `out` -> `torrentData` or append 'Out` to the other out parameters?